admin/templates/configuration/etc/log4cxx.properties

```diff
@@ -5,6 +5,6 @@
 log4j.rootLogger=DEBUG, QLogger
 log4j.appender.QLogger=org.apache.log4j.ConsoleAppender
 log4j.appender.QLogger.layout=org.apache.log4j.PatternLayout
-log4j.appender.QLogger.layout.ConversionPattern=%d{MMdd HH:mm:ss.SSS} [%t] %-5p %c{2} (%F:%L) - %m%n
+log4j.appender.QLogger.layout.ConversionPattern=%d{MMdd HH:mm:ss.SSS} [PID:%X{PID}, %t] %-5p %c{2} (%F:%L) - %m%n
```

Comment:
OK PID is optional. I think it can be useful to know if Qserv was restarted between two log lines, don't we need it?

If PID isn't in MDC, then nothing is printed here.